include/oneapi/dpl/pstl/hetero/numeric_impl_hetero.h

```diff
@@ -172,7 +172,7 @@ __pattern_transform_scan_base(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&
         oneapi::dpl::__par_backend_hetero::__parallel_transform_scan(_BackendTag{}, __policy, __buf1.all_view(),
                                                                      __buf2.all_view(), __n, __unary_op, __init,
                                                                      __binary_op, _Inclusive{})
-            .wait();
+            .__deferrable_wait();
 
         // Move data from temporary buffer into results
         oneapi::dpl::__internal::__pattern_walk2_brick(
```

Comment:
Here I have the same kind of concern as well, as another computation follows the pattern call.

Reply:
Fixed